addons/resource.language.en_gb/resources/strings.po

```diff
@@ -16541,3 +16541,13 @@ msgstr ""
 msgctxt "#38016"
 msgid "%d fps"
 msgstr ""
+
+#: system/settings/rbp.xml system/settings/imx6.xml
+msgctxt "#38120"
+msgid "Support 8 channel DTS HD audio decoding"
```

Comment:
I'm not THE entity for labels, so I only give suggestions. If you think they are fine I'll PR